src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallLibertyTask.groovy

```diff
@@ -15,10 +15,14 @@
  */
 package net.wasdev.wlp.gradle.plugins.tasks
 
+import javax.xml.parsers.*
+
 import org.gradle.api.Project
 import org.gradle.api.tasks.TaskAction
 
 class InstallLibertyTask extends AbstractTask {
+    
+    final MAVEN_REPO = 'http://repo1.maven.org/maven2/' 
```

Comment:
If adding programmatically adding the dependency causes it to be downloaded, why should we need to reference this URL? What if they are using an internal maven repo?